### ISSUE TITLE
No 2G/3G/4G toggle in Cellular Settings on Nexus 7 (Deb)

### ISSUE THREAD

**Issue by ShadowEO** [ubports/ubuntu-touch#1712]
### Description of the feature

On other LTE-supported devices, I'm seeing the 2G/3G/4G selection inside ubuntu-system-settings, however on the Nexus 7 LTE, I am not. I have attempted to add `OFONO_RIL_RAT_LTE=1` to `/etc/init/ofono.override`, but the selection didn't appear there after that change either. The tablet itself automatically connects to LTE (I see the indicator give the LTE icon) but I am unable to choose 4G if something causes it to change.

### Illustrations

- Device: Nexus 7 LTE (Deb)
- Channel: Edge
- Build: 2021-04-18

### Steps to reproduce

Simply opened to the Cellular settings

### Expected behavior

Expected to see "2G/3G/4G" option underneath the one for 2G/3G

### Actual behavior

No 2G/3G/4G option.

**Comment by ShadowEO** [comment]
I need to amend this, not sure what was going on with my initial installation however adding `OFONO_RIL_RAT_LTE=1` to the override again after reinstalling has forced the 4G selector to appear.  

It still isn't there by default, but adding the environment variable to ofono does actually make it appear now. (Pretty sure it was something in my install not letting the environment variable work when I opened this ticket)

**Comment by animeking9901** [comment]
Confirming that on edge build,Tracfone Tmobile tower sim card,2g/3g/4g is NOT a avalable option by default,which screwed me when i forced 3g due to bad coverage area,even a full wipe and reinstall did not reset the setting,gonna try that variable to fix it,and report back

**Comment by ShadowEO** [comment]
> Confirming that on edge build,Tracfone Tmobile tower sim card,2g/3g/4g is NOT a avalable option by default,which screwed me when i forced 3g due to bad coverage area,even a full wipe and reinstall did not reset the setting,gonna try that variable to fix it,and report back
> EDIT: adding the variable to the override file does add the appropriate option,maybe add it by default in the future to the override?

I agree, it should probably be in the device package for the Nexus 7 LTE.
